File: CPP/BEE-1041_Coordenadas_de_um_Ponto.cpp
/*------------------------*
| Rodrigo CavanhaMan      |
| URI 1041                |
| Coordenadas de um Ponto |
*-------------------------*/
#include<stdio.h>
using namespace std;

int main() {

	float x,y;
	scanf("%f",&x);
	scanf("%f",&y);

	if (x == 0 && y == 0)
		printf("Origem\n");
	else if(x == 0)
		printf("Eixo Y\n");
	else if(y == 0)
		printf("Eixo X\n");
	else{
		if (x > 0 && y > 0)
			printf("Q1\n");
		if (x < 0 && y > 0)
			printf("Q2\n");
		if (x < 0 && y < 0)
			printf("Q3\n");
		if (x > 0 && y < 0)
			printf("Q4\n");
	}
	return 0;
}

File: BEE-1041_Coordenadas_de_um_Ponto.java
/*-------------------*
| Rodrigo CavanhaMan |
|        IFTM        |
|      BEE 1041      |
*--------------------*/
import java.io.IOException;import java.util.Locale;
import java.util.Scanner;

public class Main {

	public static void main(String[] args)  throws IOException {
		Scanner sc = new Scanner(System.in);
		sc.useLocale(Locale.ENGLISH);
		Locale.setDefault(new Locale("en", "US"));
		
		float x,y;
		x = sc.nextFloat();
		y = sc.nextFloat();

		if (x == 0 && y == 0){
			System.out.println("Origem");
		}
		else if(x == 0){
			System.out.println("Eixo Y");
		}
		else if(y == 0){
			System.out.println("Eixo X");
		}
		else{
			if (x > 0 && y > 0){
				System.out.println("Q1");
				}
			if (x < 0 && y > 0){
				System.out.println("Q2");
				}
			if (x < 0 && y < 0){
				System.out.println("Q3");
				}
			if (x > 0 && y < 0){
				System.out.println("Q4");
				}
			}

		sc.close();
	}
}